types/shopify-buy/index.d.ts

```diff
@@ -377,12 +383,17 @@ declare namespace ShopifyBuy {
         /**
          * ID of line item variant.
          */
-        variant_id: string | number;
+        variantId: string | number;
 
         /**
          * Title of variant.
          */
-        variant_title: string;
+        variantTitle: string;
+    }
+
+    export interface LineItemAdded {
+        variantId: string | number;
+        quantity: number;
```

Comment:
I think this is missing `customAttributes: [{key: string; value: string}]`.

Reply:
Thanks, adding as optional property

Reply:
it's missing in many places :(